mdn/yari
# Set the right HTML lang attribute

Fixes #2038
Fixes #2960

How to test:

```
export CONTENT_TRANSLATED_ROOT=~/dev/MOZILLA/MDN/translated-content-rendered/files
yarn dev
```

Now view view-source:http://localhost.org:8000/zh-CN/docs/Web/JavaScript/Reference and you should see `<html lang="en">` and `<article lang="zh">`. 
And for view-source:http://localhost:5000/en-US/docs/Web/JavaScript/Reference you should see `<html lang="en">` and `<article lang="en">`.
